app-admin/etckeeper/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=etckeeper
+PKGSEC=utils
+PKGDEP="git python-3"
+PKGDES="Store /etc in git"
+ABSPLITDBG=0
```

Comment:
Missing `newline`, ditto for the two files below.

Also, is this package `noarch` (I'm guessing from `ABSPLITDBG=0`)? If so, declare `ABHOST=noarch` instead.